mubench.pipeline/tasks/implementations/check.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from typing import List
+
+from tasks.project_task import Requirement
+from tasks.project_task import ProjectTask
+from utils.shell import Shell
+
+
+class Check(ProjectTask):
+    def __init__(self):
+        super().__init__()
+
+    def start(self):
+        self._check_all_requirements()
+
+    def process_project(self, *_):
+        return self.ok()
+
+    def _check_all_requirements(self):
+        logger = logging.getLogger("requirements")
+        logger.info("Checking all requirements...")
```

Comment:
I actually removed that output there, since I figured that its not the Task Runners job to know what the tasks will do. The output now looks like this:

> Starting benchmark...
> Starting check at 14:32:51...
>     Checking all requirements...
>     All requirements satisfied. You're good to go.

We can rename the task instead, if you would like to get rid of the extra line, but to me its a bit easier to read this way.